### DISCUSSION TITLE
i didn't find any way to debug in the zed editor ?

### DISCUSSION THREAD

**Discussion by yassinekrika** [zed-industries/zed#15875] [Feature Requests]
i didn't find any way to debug in the zed editor ?

**Comment by JosephTLyons** [comment]
Hey @yassinekrika, you're right, we currently don't have a debugger or debug support of any kind at the moment. Feel free to subscribe to this issue for updates. It definitely is something we will get to.

- https://github.com/zed-industries/zed/issues/5065
